src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Removed `ServicePrincipalSecret` and `CertificatePassword` in Account of Context from display [#15427]
```

Comment:
from display or we have removed them from Azure Context psobject? I remember it is latter.

Reply:
> 

There are 2 classes that relate to Account of Context in Az.Accounts, AzureAccount is for internal use and PSAzureRmAccount is for powershell operation. The PR just removes `ServicePrincipalSecret` and `CertificatePassword` from PSAzureRmAccount.